OneCommunityGlobal/HighestGoodNetworkApp
# Harshida fix user management changes

# Description
Make new people added to system show up at the top of the user list.
1. Admin Login → Other Links → User Management
2. Newest people should show up at the top of the user list. Currently, if I add multiple people one after another, it looks like it is alphabetizing them by day and based on their first names. Newest people should always be at the top. 
3. Please also add a copy icon and ability by their first name, last name, and end date. 


<img width="551" alt="Screenshot 2023-10-07 100112" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/109878936/2bd89cd4-0e11-4929-ad8f-332786d2f336">

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin user
4. go to dashboard→ Admin Login → Other Links → User Management
5. check copy icon available by their first name, last name, and end date. (Similar to email)
6. Now create new user. 
7. Verify newest people should show up at the top of the user list. (Note: user list to be organized with latest START date at the top)

## Note:
User list to be organized with latest START date at the top.



https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/109878936/d202f1e9-589a-401c-9886-6901f1c69b0b


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/109878936/6a944238-d201-4a7c-b474-99ceebda1c65
